src/Icon/index.js

```diff
@@ -1,9 +1,9 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import PropTypes from "prop-types";
```

Comment:
why change all single quotes to double quotes?  Is the eslint config different here from other repos?